### DISCUSSION TITLE
--configmap= arg reference to argo-workflows-workflow-controller-configmap

### DISCUSSION THREAD

**Discussion by tooptoop4** [argoproj/argo-workflows#8936] [Q&A] [answered]
i see server-deployment and workflow-controller-deployment both refer to configmap using non-standard k8s way as in args in the command. this makes it less easy to refer to env variables from already existing configmap, any recommendation for this?

ie i already have proxyconfigmap with a few variables defined, i currently am repeating those variables in argo-workflows-workflow-controller-configmap

**Comment by terrytangyuan** [comment, marked as answer]
We currently only support this via arg so unfortunately that is the only way. You can use some third-party configmap sync mechanism/controller if you don't want to manually repeat the variables.
